packages/strapi-plugin-content-manager/admin/src/containers/App/index.js

```diff
@@ -41,6 +42,20 @@ class App extends React.Component {
     } else {
       this.props.loadModels();
     }
+
+    const modelName = this.props.location.pathname.split('/')[3];
```

Comment:
Is there a better way to get this value?  Maybe, we should define a new route with more parameters in the `App` container. This line of code seems pretty fragile.

Reply:
There is no better way  because I dont't have access to the `match.params` props in this container.